frontend/src/components/ComboBox.tsx

```diff
@@ -63,18 +70,71 @@ export default function ComboBox({
           display: "none",
         },
       }),
-      multiValue: (base: any) => ({
+      singleValue: (base: CSSObjectWithLabel) => ({
+        ...base,
+        color: primary,
+        fontWeight: "bold",
+        fontSize: "0.875rem",
+      }),
+      multiValue: (base: CSSObjectWithLabel) => ({
         ...base,
+        color: primary,
+        fontWeight: "bold",
+        fontSize: "0.875rem",
+        minWidth: "auto",
+        borderRadius: 8,
         flex:
           selectedMultipleOptionsValue?.length &&
           selectedMultipleOptionsValue?.length >= 2
             ? "1 0 auto"
             : "",
       }),
-
-      control: (base: CSSObjectWithLabel) => ({
+      control: (base: CSSObjectWithLabel, { isDisabled }: any) => {
+        return {
+          ...base,
+          width,
+          borderRadius: 8,
+          borderColor: !isDisabled ? primary : black,
+          "&:hover": {
+            borderColor: primary,
+            backgroundColor: `${primary}10`,
+          },
+        };
+      },
+      menu: (base: CSSObjectWithLabel) => ({
         ...base,
         width,
+        borderRadius: 8,
+        padding: "0 0.25rem",
+      }),
+      option: (base: CSSObjectWithLabel, { isSelected, data }: any) => ({
+        ...base,
+        color: isSelected ? "white" : data?.disabled ? `${primary}50` : primary,
+        fontWeight: "bold",
+        fontSize: "0.75rem",
+        lineHeight: "1.5rem",
+        padding: "0.5rem 0.75rem",
+        margin: "0.25rem 0",
```

Comment:
Lett at dette "sklir ut" fra resten om man for eksempel endrer fontstørrelser i tailwind? Kult triks du gjorde over hvor du henta ut farger fra tailwind. Om noe lignende går for dette også, hadde det vært kult. Hvis ikke går det vel fint, men man må vær litt obs på vedlikeholdet her da